File: test/end-to-end/String Indices/string indices.py
#
# Initialization
haystack = "Hello, GLS!"

# Searching
needle = "GLS"
firstIndexOf = haystack.find(needle)
secondIndexOf = haystack.find(needle, firstIndexOf + len(needle))

# Results
print("Found a first result at: {0}.".format(firstIndexOf))

if secondIndexOf != -1:
    print("Found a second result at: {0}.".format(secondIndexOf))
#

File: test/end-to-end/String Indices/string indices.rb
#
# Initialization
haystack = "Hello, GLS!"

# Searching
needle = "GLS"
firstIndexOf = haystack.index(needle)
secondIndexOf = haystack.index(needle, firstIndexOf + needle.length)

# Results
puts "Found a first result at: $0%d." % [firstIndexOf]

if secondIndexOf != nil
    puts "Found a second result at: $0%d." % [secondIndexOf]
end
#
